cg/server/api.py

```diff
@@ -498,6 +498,8 @@ def parse_orderform():
             order_parser = JsonOrderformParser()
             order_parser.parse_orderform(order_data=json_data)
         parsed_order: Orderform = order_parser.generate_orderform()
+        order_service = OrderService(db)
+        order_service.create_order(parsed_order)
```

Comment:
I am uncertain if this is the correct place to put this - this is only used to parse the orderform and return the value to the FE. I don't think an order has actually been submitted at this point, so nothing should be persisted to the database. But please correct me if I'm wrong.

Reply:
Oh, there is a `submit_order` endpoint 😅

Reply:
Moved it 👍